spec/features/homepage_spec.rb

```diff
@@ -1,12 +1,17 @@
 require 'rails_helper'
 
 describe 'Some home page sanity checks' do
-    it 'pins records to the home page map and allows users to navigate to them' do
-      Capybara.current_driver = :selenium
-      visit '/'
-      expect(page).to have_content('Browse all 5 items from 189 historical societies, libraries, archives, and cultural organizations across Minnesota')
-      find(:xpath, '//*[@id="search"]').click
-      expect(page).to have_content('A. Aberle residence, 214 South Peck Street, Fergus Falls, Minnesota')
+  before do
+    VCR.use_cassette('ingest_otter:297') do
```

Comment:
With `VCR`, HTTP requests are recorded once and "played back" on subsequent test runs, so that a network connection is not a dependency of the test suite. It mocks the HTTP responses of our external dependencies using data that was returned by them in the course of making real requests.